tests/sample_schemas/relative_ref/dereferenced_swagger.json

```diff
@@ -1,77 +1,98 @@
-                "description": "",
-                "operationId": "no_models_get"
-            }
+                "logging_info": {
+                  "type": "object"
+                }
+              },
+              "additionalProperties": false,
+              "type": "object",
+              "required": [
+                "raw_response",
+                "logging_info"
+              ]
+            },
+            "description": "Return a standard_response"
+          }
         },
-        "/sample/{path_arg}/resource": {
-            "get": {
-                "responses": {
-                    "200": {
-                        "description": "Return a standard_response",
-                        "schema": {
-                            "type": "object",
-                            "required": [
-                                "raw_response",
-                                "logging_info"
-                            ],
-                            "additionalProperties": false,
-                            "properties": {
-                                "raw_response": {
-                                    "type": "string"
-                                },
-                                "logging_info": {
-                                    "type": "object"
-                                }
-                            }
-                        }
-                    }
+        "operationId": "standard",
+        "description": "",
+        "parameters": [
+          {
+            "$ref": "#/definitions/file.parameters..common.json|..definitions..path_arg"
+          },
+          {
+            "$ref": "#/definitions/file.paths..common.json|..definitions..path_args"
+          }
+        ]
+      }
+    },
+    "/no_models": {
+      "get": {
+        "responses": {
+          "200": {
+            "schema": {
+              "properties": {
+                "raw_response": {
+                  "type": "string"
                 },
-                "description": "",
-                "operationId": "standard",
-                "parameters": [
-                    {
-                        "in": "path",
-                        "name": "path_arg",
-                        "required": true,
-                        "type": "string",
-                        "enum": ["path_arg1", "path_arg2"]
-                    }
-                ]
-            }
-        }
+                "logging_info": {
+                  "type": "object"
+                }
+              },
+              "additionalProperties": false,
+              "type": "object",
+              "required": [
+                "raw_response",
+                "logging_info"
+              ]
+            },
+            "description": "Return a standard_response"
+          }
+        },
+        "operationId": "no_models_get",
+        "description": ""
+      }
+    }
+  },
+  "definitions": {
+    "file.paths..common.json|..definitions..path_args": {
```

Comment:
I'm not sure what you're doing conforms to the Swagger spec: What you're describing here is a Parameter object, while according to the spec `definitions` needs to be a name -> Schema object mapping. This might cause issues with tools that are not as lenient as bravado_core / swagger_schema_validator.